qml/Components/KeyboardShortcutsOverlay.qml

```diff
@@ -198,30 +198,6 @@ Rectangle {
                     wrapMode: Text.Wrap
                     Layout.maximumWidth: maxTextSize
                 }
-
-
-                // Scopes section
-                Item { Layout.columnSpan: 2; height: units.gu(2) }
-                Label {
-                    Layout.columnSpan: 2
-                    text: i18n.tr("Scopes")
-                    font.weight: Font.Light
-                    color: theme.palette.normal.baseText
-                    lineHeight: 1.3
-                }
-
-                Label {
-                    text: i18n.tr("Super (Tap)")
```

Comment:
Super (Tap) opens the side "applauncher" maybe we should keep this but change from scope to App Launcher?

Reply:
I don't think we need to include it (and I'd prefer to not introduce new strings here). If we feel it needs to be included, we can add it in a later commit.